packages/gatsby-theme-nodeschool/src/locales/en/translation.json

```diff
@@ -0,0 +1,77 @@
+{
+  "Welcome to": "Welcome to",
+  "Community": "Community",
+  "The NodeSchool community is": "The NodeSchool community is",
+  "open to anyone": "open to anyone",
+  "Whether you're able to attend an event or not, you can join the discussion around learning Node": "Whether you're able to attend an event or not, you can join the discussion around learning Node",
+  "in our Slack channel": "in our Slack channel",
+  "by following us on Twitter": "by following us on Twitter",
+  "or": "or",
```

Comment:
So from past jobs concatinating translations together doesn't really work well. Do you know if it has placeholders/variables support?

Reply:
I think yes, https://www.i18next.com/translation-function/interpolation

I didn't it because I was rushed.

Could we do it together in another merge request dedicated to organize it with interpolation (placeholders/variables)?